tests/expr/fbinary/test-truediv.py

```diff
@@ -116,21 +126,21 @@ def test_div_floats_random(seed):
 def test_mul_booleans_integers_floats_random(seed):
     random.seed(seed)
     n = 1000
-    src1 = [random.randint(-100, 100) for _ in range(n)]
+    src1 = [random.randint(1, 100) for _ in range(n)]
     src2 = [random.randint(0, 1) for _ in range(n)]
-    src3 = [random.random() * 1000 - 500 for _ in range(n)]
+    src3 = [random.random() * 1000 + 1 for _ in range(n)]
```

Comment:
Perhaps, instead of narrowing this test down to positive numbers only, we simply create a helper function `dt_div(num, denom)` that will implement the datatable logic in terms of division and will return one of the following `num/denom` `math.inf`, `-math.inf` or `None`. Then, this function could be used to fill out the `EXP` frame below. It could also be used in the `test_div_integers()` test.

Reply:
Done